nodejs/week1/week1homework/src/bakckend/routes/meal.js

```diff
@@ -0,0 +1,20 @@
+
+function getAMeal(){
+    
+  mealArr=require("../data/meals.json");
+  reviewArr=require("../data/reviews.json");
+
+  const reviewMeal= mealArr.map((meal)=>{
+      meal.reviews = reviewArr.filter((review) => review.mealId === meal.id);
+      return meal;
+  })
+ const singleMeal=reviewMeal[1];
+ return singleMeal;
+}
```

Comment:
Use const to declare variables
Fix formatting in this file. 
You also have the opportunity to reuse `getMealsWithReviews`.